### ISSUE TITLE
Add global parameter for bootstrap version 2/3/4 that is loaded

### ISSUE THREAD

**Issue by ggppdk** [FLEXIcontent/flexicontent-cck#413]
Some template makers try to use template overrides that use bootstrap 3

For frontend display
- we can add a configuration option to define the bootstrap version that is active in frontend,
thus using use bootstrap 2/3/4 CSS classes and other appropriate layout that is suitable for bootstrap version that is active

FLEXIcontent fields , item form, and template code will be updated slowly to support this

**Comment by micker** [comment]
yes ... maybe adding other framwork compatibility like warp, t3( bootstrap3), etc
